### ISSUE TITLE
Google map content is not loading

### ISSUE THREAD

**Issue by sunilkumarbm** [angular-ui/angular-google-maps#1947]
I'm facing a weird issue with google maps.

As shown in the image below, the container meant to load the map content is not showing the map. It is only showing the map controls (map type, zoom controls) and google logo.

This issue started on Aug 31. Till then it was working fine.

I'm using 2.3.1 version of angular google maps and angular.js version 1.5.0. Please suggest on how to resolve this issue.

![angular gmap](https://cloud.githubusercontent.com/assets/9527600/18623482/3f5092da-7e5b-11e6-98dc-1ff53d214aa1.png)

**Comment by returnvoid** [comment]
Same problem here. In my case it happens with templates. If I load the page as initial, it works fine. But if I follow the flow of my app, when I come back using $location.path the maps is empty.

**Comment by nmccready** [comment]
I think this might be because Google maps api no longer likes to allow you to load the map asynchronously without an API key. Try sticking the google maps script url synchronously into the html and the error might disappear.

However, this doesn't really resolve the problem. I think to solve it for now you would want to decorate the api loader services, factory and or provider to prevent it from loading the sdk async.

https://github.com/angular-ui/angular-google-maps/blob/master/src/coffee/providers/map-loader.coffee#L3-L113

Basically you'll need to trip this on whether you have a key or not. This is basically compared to development mode vs production.

**Comment by sunilkumarbm** [comment]
I was able to resolve this issues by using new version of google maps. Somehow, the version which I was using stopped working on one fine day.
